tests/phpunit/tests/term/wpTerm.php

```diff
@@ -89,4 +101,36 @@ public function test_get_instance_should_respect_taxonomy_when_term_id_is_found_
 		$found = WP_Term::get_instance( self::$term_id, 'wptests_tax2' );
 		$this->assertFalse( $found );
 	}
+
+	/**
+	 * @ticket 58329
+	 */
+	public function test_get_instance_no_sanitize() {
```

Comment:
Could we consider adding a test with a data provider? In the data provider, I've included a test case, and if it's not necessary, we can remove it. Thank you.

```
	/**
	 * @ticket 58329
	 *
	 * @dataProvider data_get_instance_with_sanitize
	 *
	 * @param string|false $sanitize Sanitize term fields.
	 */
	public function test_get_instance_with_sanitize( $sanitize ) {
		$obj           = new stdClass();
		$obj->taxonomy = 'wptests_tax';
		foreach ( self::$int_fields as $int_field ) {
			$obj->$int_field = 'test';
		}
		wp_cache_add( 999, $obj, 'terms' );
		$term = WP_Term::get_instance( 999, 'wptests_tax', $sanitize );
		foreach ( self::$int_fields as $int_field ) {
			$this->assertIsInt( $term->$int_field );
		}
	}

	/**
	 * Data provider.
	 *
	 * @return array[]
	 */
	public function data_get_instance_with_sanitize() {
		return array(
			'check false value for sanitize'      => array( false ),
			'check empty value for sanitize'      => array( '' ),
			'check raw value for sanitize'        => array( 'raw' ),
			'check any string value for sanitize' => array( 'test' ),
		);
	}
```